applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h

```diff
@@ -147,15 +141,17 @@ class DistanceModificationProcess : public Process
     ///@name Member Variables
     ///@{
 
-    ModelPart&                                              mrModelPart;
-    double                                                 mFactorCoeff;
-    double                                           mDistanceThreshold;
-    bool                                               mCheckAtEachStep;
-    bool                                           mNegElemDeactivation;
-    bool                                       mRecoverOriginalDistance;
-    bool                                      mAvoidAlmostEmptyElements;
-    std::vector<std::vector<unsigned int>>        mModifiedDistancesIDs;
-    std::vector<std::vector<double>>           mModifiedDistancesValues;
+    ModelPart&                                                    mrModelPart;
+    double                                                 mDistanceThreshold;
+    bool                                                          mIsModified;
+    bool                                                  mContinuousDistance;
+    bool                                                     mCheckAtEachStep;
+    bool                                                 mNegElemDeactivation;
+    bool                                             mRecoverOriginalDistance;
+    bool                                            mAvoidAlmostEmptyElements;
+    std::vector<std::vector<unsigned int>>              mModifiedDistancesIDs;
+    std::vector<std::vector<double>>                 mModifiedDistancesValues;
```

Comment:
this tends to be very slow. do you really need it?

Reply:
Just in case you need to keep the original distance because any reason (we've never used it but I did in case it is needed).